client/src/app/components/Sidebar/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default () => {
+  return (
+    <div className='flex flex-col py-4 items-center'>
+      <Link to='/newgame'>
+        <CirclePlusIcon className='mb-4 text-gray-300 hover:text-white w-8' />
+      </Link>
+      <Link to='/home'>
+        <ListIcon className='mb-4 text-gray-300 hover:text-white w-8' />
+      </Link>
+      <Link to='/account'>
+        <UserIcon className='text-gray-300 hover:text-white w-8' />
+      </Link>
+    </div>
+  );
+};
+
+const CirclePlusIcon = props => (
```

Comment:
If we can't use [the sprite example](https://github.com/tabler/tabler-icons#svg-sprite), then I think @AaronRMichael would suggest that we move each icon into its own file